ingest/src/main/scala/hydra.ingest/http/HealthEndpoint.scala

```diff
@@ -20,7 +28,20 @@ object HealthEndpoint extends RouteSupport with DefaultJsonProtocol with SprayJs
             pathEndOrSingleSlash {
               get {
                 addHttpMetric("", StatusCodes.OK, "/health", startTime, "GET")
-                complete(BuildInfo.toJson)
+                onComplete(Futurable[F].unsafeToFuture(cga.consumerGroupIsActive())) {
+                  case Failure(exception) =>
+                    addHttpMetric("",StatusCodes.InternalServerError,"/v2/tags", startTime, method.value, error = Some(exception.getMessage))
```

Comment:
Is the error supposed to have "v2/tags"? Or "v2/health"?

Reply:
Good call, I will change those. Can you tell where my copy pasta came from?